sale_terms_template/README.rst

```diff
@@ -0,0 +1,73 @@
+===================
+Sale terms template
+===================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/14.0/sale_terms_template
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-14-0/sale-workflow-14-0-sale_terms_template
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module add terms and condition template and change existing terms and condition
+(`sale_order.note`) field type from `Text` to `Html`.
```

Comment:
as the field is ``note``, I think the technical name of module should use note also. like ``sale_order_note_template``.

Reply:
Do you suggest to use a technical name `sale_order_note_template` different from the module name `Sale terms template` in order to let functional users to find the module in apps store and let technical guys understand which fields is used ?

Reply:
My proposal is to use the same words as Odoo sale module, so : 
 
- technical module name ``sale_order_note_template``.
- Friendly module name ``Sale Orders Terms and conditions Templates``

https://github.com/odoo/odoo/blob/12.0/addons/sale/models/sale.py#L202

Reply:
ok, thanks let's do that :)

Reply:
I've renamed the module as suggest !